model/Image.php

```diff
@@ -123,25 +123,13 @@ public function getCMSFields() {
 	}
 
 	/**
-	 * An image exists if it has a filename.
-	 * Does not do any filesystem checks.
```

Comment:
This to me does feel like this is a change in behaviour... I think it's worth doing a bit of digging into this method to see where it was introduced, and why it explicitly doesn't do a filesystem check. If it turns out that use case is no longer valid then maybe it would be ok to merge this into 3.1, but maybe even then it'd be safer to target the 3 branch.